src/main/java/AirBorder.java

```diff
@@ -0,0 +1,77 @@
+import java.util.Scanner;
+
+
+    public class AirBorder {
+        public static void main(String[] args) {
+            Scanner scanner = new Scanner(System.in);
+            Task[] tasks = new Task[100];  //
+            int taskCount = 0;
+
+            System.out.println("____________________________________________________________");
+            System.out.println(" Hello! I'm AirBorder");
+            System.out.println(" What can I do for you?");
+            System.out.println("____________________________________________________________");
+
+            while (true) {
+                String userInput = scanner.nextLine().trim().toLowerCase();
+
+                if (userInput.equals("bye")) {
+                    System.out.println("____________________________________________________________");
+                    System.out.println(" Bye. Hope to see you again soon!");
+                    System.out.println("____________________________________________________________");
+                    break;
+                } else if (userInput.equals("list")) {
+                    System.out.println("____________________________________________________________");
+                    System.out.println(" Here are the tasks in your list:");
+                    if (taskCount == 0) {
+                        System.out.println(" No tasks added.");
+                    } else {
+                        for (int i = 0; i < taskCount; i++) {
+                            System.out.println(" " + (i + 1) + ". " + tasks[i]);
+                        }
+                    }
+                    System.out.println("____________________________________________________________");
+                } else if (userInput.startsWith("mark ")) {
+                    int taskNumber = Integer.parseInt(userInput.split(" ")[1]) - 1;
```

Comment:
Could you think of another way of writing this?

Reply:
String[] commandParts = userInput.split(" ");
int taskNumber = Integer.parseInt(commandParts[1]) - 1;  
Does this approach look better?

Reply:
This change has been made redundant due to changes made in future increments